test/requests/document_symbol_expectations_test.rb

```diff
@@ -24,6 +24,28 @@ def test_document_symbol_addons
     end
   end
 
+  def run_expectations(source)
```

Comment:
It looks like the document symbol request is still instantiated the same way it was before. Why is overriding `run_expectations` necessary?